app/components/ResourcePage.tsx

```diff
@@ -0,0 +1,53 @@
+'use client';
+import { redirect } from 'next/navigation';
+import Link from 'next/link';
+import ResourcesSection from '@/app/components/ResourcesSection';
+// import ResourcesSidebar from '@/app/components/ResourcesSideBar';
+import translations from '@/public/locales/en.json';
+import { useTranslation } from 'react-i18next';
+import { ResourceCategories, Subcategory } from '@/app/constants/interfaces';
+
+export default function ResourcePage({ slug }: { slug: string }) {
+  const { t } = useTranslation();
+
+  const categories = translations.resources
+    .categories as unknown as ResourceCategories;
+
+  const category = categories[slug];
+
+  if (!category) {
+    redirect('/');
+  }
+
+  return (
+    <main className='text-white '>
+      <Link href='/' className='hover:underline'>
+        {t('resources.backToResources')}
+      </Link>
+      <section className='flex flex-row py-5'>
+        <div id='sidebar' className='basis-5/12'>
+          <h1 className='text-2xl font-bold'>
+            {t(`resources.categories.${slug}.name`)}
+          </h1>
+          {/* <ResourcesSidebar resources={category.resources} /> */}
```

Comment:
Need to refactor this sidebar to work the same way as the other part of the page using the new data structure.